cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -1246,6 +1252,63 @@ public void testNoKafkaCluster(VertxTestContext context) {
                 })));
     }
 
+    /**
+     * Tests the transition from 'New' to 'NotReady' due to missing Kafka cluster
+     *
+     * 1. A new KafkaRebalance resource is created; it is in the New state
+     * 2. The operator checks that the Kafka cluster specified in the KafkaRebalance resource (via label) doesn't exist
+     * 4. The KafkaRebalance resource moves to NotReady state
+     */
```

Comment:
AFAICS the Javadoc doesn't reflect the test case, or?

Reply:
Should be fixed.